django_mongodb/query.py

```diff
@@ -50,19 +50,23 @@ def __init__(self, compiler):
         self.collection = self.compiler.collection
         self.collection_name = self.compiler.collection_name
         self.mongo_query = getattr(compiler.query, "raw_query", {})
-        self.subquery = None
+        self.subqueries = None
         self.lookup_pipeline = None
         self.project_fields = None
         self.aggregation_pipeline = compiler.aggregation_pipeline
         self.extra_fields = None
         self.combinator_pipeline = None
+        # Stage that encapsulates the pipeline for performing a nested subquery within a $lookup.
+        self.subquery_lookup = None
 
     def __repr__(self):
         return f"<MongoQuery: {self.mongo_query!r} ORDER {self.ordering!r}>"
 
     @wrap_database_errors
     def delete(self):
         """Execute a delete query."""
+        if self.compiler.subqueries:
+            raise NotSupportedError("Subqueries are not supported in delete operations.")
```

Comment:
How about "Cannot use QuerySet.delete() when a subquery is required."? (And this probably deserves e a mention in README?)

Reply:
Could be, but I think is contained in:`QuerySet.delete()` and `update()` do not support queries that span multiple  collections.